js/tests/instantiate.test.ts

```diff
@@ -12,83 +14,23 @@ export function tests() {
         }
 
         canInstantiate(
-            "string        | no default program",
-            new Nautilus(CONNECTION, PROGRAM_ID_STRING), 
+            "string        | no programs",
```

Comment:
this says "no programs" but you're passing in a program ID as a string still

Reply:
Some of the confusion in the tests comes from the fact that I struggled to understand what your intent for the constructor parameters were.
From my understanding, you could pass a list of programs (e.g. Token Metadata, Token2022, etc) that Nautilus will use, or take the default.

In this case, it says "no programs" because no program is overriden: there is no `programs` parameter passed.